src/target/riscv/riscv-013_reg.c

```diff
@@ -13,73 +13,454 @@
 #include "debug_defines.h"
 #include <helper/time_support.h>
 
-static int riscv013_reg_get(struct reg *reg)
+/* Start of register fetch/send definitions.
+ * - "reg_fetcher" (named "fetch_*") -- loads the value from target into reg
+ *   cache entry. Modifyes only "reg->value".
```

Comment:
Pedantic detail - typo

```suggestion
 *   cache entry. Modifies only "reg->value".
```

Reply:
Addressed.